integration.md

```diff
@@ -33,7 +33,7 @@ It means that the contributors have spent a while thinking through the APIs from
 
 # Supported Tracer Implementations
 
-## Appdash
+## Appdash ([https://github.com/sourcegraph/appdash/blob/master/README.md](github.com/sourcegraph/appdash))
```

Comment:
You swapped link text and URL here and for all the other ones, too. The correct syntax is `[text](http://...)`